utils/fetch_external_bangs.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+
```

Comment:
Since there is a shebang, I would suggest to set file mode to 755.

OT: there are some trailing whitespace which can be dropped:

```diff
@@ -22,7 +22,7 @@ RE_BANG_VERSION = re.compile('\/bang\.v([0-9]+)\.js')
 def get_bang_url():
     response = requests.get(URL_BV1)
     response.raise_for_status()
-    
+
     r = RE_BANG_VERSION.findall(response.text)
     return f'https://duckduckgo.com/bang.v{r[0]}.js', r[0]
 
@@ -36,7 +36,7 @@ def fetch_bangs(url):
 def merge_when_no_leaf(node):
     """Minimize the number of nodes
 
-    A -> B -> C 
+    A -> B -> C
     B is child of A
```

Reply:
:+1: for making it executable.

Reply:
It is now executable, and pylint checks this file.